docs/examples/tutorial/string/cclass.py

```diff
@@ -0,0 +1,13 @@
+import cython
+
+@cython.cclass
+class A:
+    cython.declare(a=cython.int, b=cython.int)
+    c = cython.declare(cython.int, visibility='public')
+    d = cython.declare(cython.int, 5)  # private by default.
+    e = cython.declare(cython.int, 3, visibility='readonly')

```

Comment:
This is actually mishandled by Cython. Default arguments are not supported here. Please set them in `__init__()` instead, via default keyword arguments. (Great to have these examples tested!)